internal/executor/instance/instance.go

```diff
@@ -126,12 +133,42 @@ func (inst *Instance) Run(ctx context.Context, config *RunConfig) error {
 			Memory:   int64(inst.memory * mebi),
 		},
 	}
+
+	// In case the additional containers are used, tell the agent to wait for them
+	if len(inst.additionalContainers) > 0 {
+		var ports []string
+		for _, additionalContainer := range inst.additionalContainers {
+			ports = append(ports, strconv.FormatUint(uint64(additionalContainer.ContainerPort), 10))
+		}
+		commaDelimitedPorts := strings.Join(ports, ",")
+		containerConfig.Env = append(containerConfig.Env, "CIRRUS_PORTS_WAIT_FOR="+commaDelimitedPorts)
+	}
+
 	cont, err := cli.ContainerCreate(ctx, &containerConfig, &hostConfig, nil, "")
 	if err != nil {
 		return err
 	}
 
+	// Start additional containers (if any)
+	var acWG sync.WaitGroup
```

Comment:
Maybe use a more descriptive name like `additionalContainersWaitGroup` or `additionalContainersWG`?

Reply:
Looks to verbose to me. Do you want to rename `acCtx` and `acCancel` too?

Reply:
Yeah, this is by JVM background talks 😅 But Go overall is very verbose! In general if I see a one or two letters variable beside `i` and `j` I want to understand what it is without looking up the declaration. `WG` seems like a convention thought but personally I'd name it `additionalContainersWaitGroup`.

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/15/commits/f0d70b442879f54bfcae1e3c1b94fde9a1c34b4e.